src/components/contentWrapper/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styled from 'styled-components'
+import { breakpoints } from '../../utils'
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+
+  ${breakpoints.mobileSmall} {
+    width: ${({ fullWidth }) => (fullWidth ? '100vw' : 'calc(100vw - 48px)')};
+  }
+
+  ${breakpoints.mobileMedium} {
+    width: ${({ fullWidth }) => (fullWidth ? '100vw' : 'calc(100vw - 40px)')};
+  }
+
+  ${breakpoints.tablet} {
+    width: ${({ fullWidth }) => (fullWidth ? '100vw' : 'calc(100vw - 80px)')};
+  }
+`
+
+const ContentWrapper = ({ children, fullWidth }) =>
+  fullWidth ? (
+    <Wrapper fullWidth>{children}</Wrapper>
+  ) : (
+    <Wrapper>{children}</Wrapper>
+  )
```

Comment:
```suggestion
const ContentWrapper = ({ children, fullWidth }) =>(
    <Wrapper fullWidth={fullWidth}>{children}</Wrapper>
  )

Reply:
Because it will have a default prop u don't have to worry about the initial value and if it was undefined that's also a falsy value so it's alright